src/main/java/teammates/common/datatransfer/attributes/StudentProfileAttributes.java

```diff
@@ -25,7 +25,13 @@
     public String email;
     public String institute;
     public String nationality;
-    public String gender; // only accepts "male", "female" or "other"
+
+    public enum Gender {
+    male, female, other, none
```

Comment:
- Enums should be all uppercase
- Why is there a need for "none"?
- This enum is duplicated three times. Do you not think it should be extracted to a more appropriate place instead? Hint: where is the place which specifies that "male", "female", and "other" are the only acceptable types for gender?